buildSrc/src/main/kotlin/plugin.gradle.kts

```diff
@@ -1,10 +1,8 @@
 plugins {
 	id("dev.yumi.gradle.licenser")
 
-	java
-
-	kotlin("jvm")
 	kotlin("plugin.serialization")
+	com.github.johnrengelman.shadow
```

Comment:
Isn't this supposed to be... an ID? What's up with this? lmao

Reply:
An id just without the string

Reply:
Let's perhaps stick with the convention and use `id()` here.

Reply:
This is the convention for plugins already in class path, that's why the accessor was generated

Reply:
I was referring to the project convention, which is to use `id()`.